pyat/at/physics/nonlinear.py

```diff
@@ -37,49 +42,65 @@ def tunes_vs_amp(ring: Lattice, amp: Optional[Sequence[float]] = None,
     """
 
     def _gen_part(ring, amp, dim, orbit, ld, nturns):
-        invx = numpy.array([[1/numpy.sqrt(ld['beta'][0]), 0],
-                            [ld['alpha'][0]/numpy.sqrt(ld['beta'][0]),
-                            numpy.sqrt(ld['beta'][0])]])
-
-        invy = numpy.array([[1/numpy.sqrt(ld['beta'][1]), 0],
-                            [ld['alpha'][1]/numpy.sqrt(ld['beta'][1]),
-                            numpy.sqrt(ld['beta'][1])]])
-        part = numpy.array([orbit, ] * len(amp)).T + 1.0e-6
+        invx = np.array(
+            [
+                [1 / np.sqrt(ld["beta"][0]), 0],
+                [ld["alpha"][0] / np.sqrt(ld["beta"][0]), np.sqrt(ld["beta"][0])],
+            ]
+        )
+
+        invy = np.array(
+            [
+                [1 / np.sqrt(ld["beta"][1]), 0],
+                [ld["alpha"][1] / np.sqrt(ld["beta"][1]), np.sqrt(ld["beta"][1])],
+            ]
+        )
+        part = (
+            np.array(
+                [
+                    orbit,
+                ]
+                * len(amp)
+            ).T
+            + 1.0e-6
+        )
```

Comment:
Surprising result !

Apparently this is due to the comma after `orbit`. Why this comma ? in a list, is useless, but I don't understand the whole statement…

Reply:
Yes it was the comma. You are right, I am not so sure why this 1.0e-6 was added to all coordinates and how its value was determined, maybe to be sure you get some signal from the cross terms, @lcarver any idea?

At least its value could be determined to adapt to the beta functions....